tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.js

```diff
@@ -0,0 +1,142 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const importPage = require('@pages/BO/advancedParameters/import');
+const customersPage = require('@pages/BO/customers');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_advancedParameters_import_importFile';
+
+// Variables
+let browserContext;
+let page;
+
+// Variable Used in the download / rename / upload test
+let sampleFilePath;
+const sampleFile = {
+  args:
+    {
+      dropdownValue: 'Customers',
+      type: 'customers_import',
```

Comment:
You're using this value as the name everywhere, so either add a `name` key with the correct value or change this key name, it will be more relevant.

Reply:
done